grommet/grommet
# fix: starRating - form value not synched

#### What does this PR do?
fix: starRating - form value not synched
#### Where should the reviewer start?

#### What testing has been done on this PR?
Star Rating Component
#### How should this be manually tested?
1. Manually clicking the start icon to ensure the expected behavior and set default value.
2. Local Jest tests.

#### Do Jest tests follow these best practices?


#### Any background context you want to provide?

#### What are the relevant issues?
#7071
#### Screenshots (if appropriate)
<img width="1460" alt="Screenshot 2024-01-04 at 12 37 47 PM" src="https://github.com/grommet/grommet/assets/134603758/c1f06b4a-dc55-49bc-b45b-9a2f8e69bd26">



#### Do the grommet docs need to be updated?

#### Should this PR be mentioned in the release notes?

#### Is this change backwards compatible or is it a breaking change?
No
